### PR TITLE
Multi-page layout

### PR DESCRIPTION
**Pull request by tony-ist**
This is an attempt to fix multi-page resumes.

**TL;DR**: Do not split companies and sections other than work experience when moving them to a new page.

My idea behind this is that work experience is usually the longest section in the resume, so we only move its subsections (companies) to a new page when it does not fit. Other sections are relatively small so we can move the whole section to a new page when it does not fit.

Screenshots for reference: 
1. When work experience does not fit in 1 page:  
Screenshot 2023-09-07 at 13 49 37  

2. When other sections do not fit in 1 page:  
Screenshot 2023-09-07 at 13 49 51  

3. Possible problem when the work experience heading gets split on the second page. It can only occur when people move work experience to be not the first section. I do not think this is critical:  
Screenshot 2023-09-07 at 13 48 58  

4. Another problem is that we do not split anything into a new page on live preview (in production without the debug flag set). The best solution in my opinion would be to introduce a multi-page live preview. But that is out of the scope of this task. One possible change would be to cut out of the live preview all the content that should be on the second page in the resulting PDF. I am not sure about it though.

Let me know what you think about this, I am open to suggestions.

### CODE REVIEW

**Comment by rdleon** [comment]
Any news on this? I would love to have this feature.

**Review by rdleon — COMMENTED** [review]
Sees like a clean an easy fix.

**Comment by TabTwo** [comment]
Any news on this?

**Comment by goravseth** [comment]
Would love to see this implemented.

**Comment by espetro** [comment]
Hi 👋🏼  can we push this forward?
